distribution/lib/Standard/Google_Api/0.0.0-dev/src/Google_Credential.enso

```diff
@@ -13,3 +13,7 @@ type Google_Credential
     ## ICON key
        Access using the defaults provided by the environment variables.
     Default
+
+    ## ICON key
+       Feeding this into supporting components will give sample data for that commponent.
+    Sample
```

Comment:
I'm wondering if it wouldn't be better to have a separate type (e.g. `Google_Sample_Data_Source`) and if a method supports the 'sample' mode, it could take `credentials : Google_Credential | Google_Sample_Data_Source` clearly indicating that either is accepted.

This way, we can easily indicate in the type if the sample mode is available or not. 

In the current state, it will only be seen at runtime.

Reply:
Yeah. It's a fair suggestion. Can't decide if it is better or not. I think generally this will only be used in workflows that we have built as samples so it's probably unlikely that a user will actually build a workflow using it.

Think I'm going to merge this one as is for now.

Reply:
> so it's probably unlikely that a user will actually build a workflow using it.

I guess that suggests that keeping it separate may be a good idea.